packages/app/src/entities/token/hooks/use-get-symbol-by-token.ts

```diff
@@ -0,0 +1,7 @@
+import { SupportedToken, tokenContracts } from '@/shared/stellar/constants/tokens'
+import { useTokenCache } from '@/entities/token/context/hooks'
```

Comment:
```suggestion
import { useTokenCache } from '../context/hooks'
```
This way it shows that we use an item from the same entity.

Reply:
Thx, edited